src/cbl-mariner/manifest.json

```
@@ -16,6 +16,19 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/helix/amd64",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "cbl-mariner-2.0-helix$(FloatingTagSuffix)": {}
```

```suggestion
                "cbl-mariner-2.0-helix-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
                "cbl-mariner-2.0-helix-amd64$(FloatingTagSuffix)": {}
```

We really should be using architecture-specific tags in case there's ever a need for Arm architecture.

---

make sense. updated.